### PR TITLE
[DEBUG] Added api to export current graph layout

### PR DESCRIPTION
**Pull request by mm-near**
This will be later used by different debug tools.

### CODE REVIEW

File: core/primitives/src/views.rs [hunk 1]
```diff
@@ -391,6 +391,18 @@ pub struct PeerStoreView {
...
+pub struct EdgeView {
+    pub peer0: PeerId,
+    pub peer1: PeerId,
+    pub nonce: u64,
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub struct NetworkGraphView {
+    pub edges: Vec<EdgeView>,
+}
```
[the remark this answers is not in the document]
Comment (pompon0) [reply core/primitives/src/views.rs:404]: @mm-near have you tested how long does it take to dump a large graph (mainnet/testnet) in practice?

[the remark this answers is not in the document]
Comment (mm-near) [reply core/primitives/src/views.rs:404]: mainnet: 31 MB.. - takes around 11 seconds to download :-|

[the remark this answers is not in the document]
Comment (matklad) [reply core/primitives/src/views.rs:404]: Do we use any compression middle wares for our HTTP APIs? Might be a quick win if we don't

**Review by pompon0 — APPROVED** [review]
LGTM, as long as the concerns raised in the comments are resolved.

File: core/primitives/src/views.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (mm-near) [reply core/primitives/src/views.rs:404]: yes -that's why this (and other) endpoints are flag-disabled by default.

**Pull request merged**